backend/app/app.config.js

```diff
@@ -8,4 +8,5 @@ module.exports = {
   CUSTOM_STYLE: process.env.CUSTOM_STYLE || "",
   HOST: "0.0.0.0",
   PORT: process.env.PORT || 8080,
+  WHITELIST_DOMAINS: process.env.WHITELIST_DOMAINS || "",
```

Comment:
```suggestion
  WHITELIST_DOMAINS: process.env.WHITELIST_DOMAINS || [],
```

To keep the same data structure I recommend the use of an empty list as default.